atomicapp/providers/openshift.py

```diff
@@ -332,10 +333,10 @@ class OpenShiftProvider(Provider):
     provider_ca = None
 
     # Parsed artifacts. Key is kind of artifacts. Value is list of artifacts.
-    openshift_artifacts = {}
+    openshift_artifacts = defaultdict(list)
 
     def init(self):
-        self.openshift_artifacts = {}
+        self.openshift_artifacts = defaultdict(list)
```

Comment:
That looks strange I agree,  but it seems that I had reason for this. https://github.com/projectatomic/atomicapp/commit/c5e6f27c76823343351a3c048f5efde13e86f44d.  But to be honest I don't remember more about this, It is almost 6 months ago